### PR TITLE
#102 sidebar links review

### PR DESCRIPTION
**Pull request by hbapte**
# Description
PR  for removing unused and duplicate sider bar links

# PR Tracker (Issue Number and Link)
Issue [102](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/102)

# Testing
- On Hosted Version: Go to `vercel bot` below and click to [Visit Preview](https://atlp-devpulse-fn-git-sidebar-link-review-devpulse.vercel.app/)
- On Local version: start the server 
- Login with admin credentials and see the sidebar links

# Screenshots
sCREEN

### CODE REVIEW

**Review by robsdagreat — CHANGES_REQUESTED** [review]
Consider resolving the errors and also pull the latest changes for the code climate failure to be resolved

**Review by robsdagreat — APPROVED** [review]
Well done @hbapte !

**Comment by MnorbertVii** [comment]
well done @hbapte

**Comment by MnorbertVii** [comment]
consider informing the one you requested for a review , an approval for the PR to be merged.

**Review by niyibi250 — APPROVED** [review]

**Review by Ishimwe7 — APPROVED** [review]
well done @hbapte

**Review by tysepa — APPROVED** [review]
Make it clear by removing that code  in comment

**Review by mugishaj092 — APPROVED** [review]

File: src/components/sidebar/sidebarItems.tsx [hunk 1]
```diff
@@ -7,11 +8,11 @@ export const sidebarItems1 = [
     icon: <Icon icon="fontisto:pie-chart-1"></Icon>,
     title: 'Dashboard',
   },
-  {
-    path: '/organisations',
-    icon: <Icon icon="fluent:people-team-20-filled"></Icon>,
```
Comment (mugishaj092) [inline src/components/sidebar/sidebarItems.tsx:12]: remove those comments and rebase

**Review by mugishaj092 — APPROVED** [review]

File: src/components/sidebar/sidebarItems.tsx [hunk 2]
```diff
@@ -27,16 +28,16 @@ export const sidebarItems1 = [
     icon: <Icon icon="ic:round-maps-home-work"></Icon>,
     title: 'View Job Post',
   },
-  {
-    path: '/admins',
-    icon: <Icon icon="ic:round-people"></Icon>,
-    title: 'Admins',
```
Comment (mugishaj092) [inline src/components/sidebar/sidebarItems.tsx:33]: remove those comments and rebase

**Review by mugishaj092 — CHANGES_REQUESTED** [review]

File: src/components/sidebar/sidebarItems.tsx (hunk 1 above)
Comment (hbapte) [reply src/components/sidebar/sidebarItems.tsx:12]: solved

**Comment by hbapte** [comment]
> Make it clear by removing that code in comment

Done

**Review by mugishaj092 — APPROVED** [review]

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**